userspace/openpilot_python_dependencies.sh

```diff
@@ -1,21 +1,14 @@
 #!/bin/bash -e
 
-echo "Installing python for openpilot"
+echo "installing uv..."
 
-# Install pyenv
-export PYENV_ROOT="/usr/local/pyenv"
-curl https://pyenv.run | bash
-export PATH="$PYENV_ROOT/bin:$PATH"
-eval "$(pyenv init -)"
+export XDG_DATA_HOME="/usr/local"
```

Comment:
@adeebshihadeh The XDG_DATA_HOME only sets the uv venv location (usr/local) otherwise the default location would fall to current directory (.venv).

Reply:
Here it doesn't affect anything except providing the installation part for the venv. But we should avoid usage it /etc/profile as this will make it Into the envvars and might have some influence.